rust/crd/src/lib.rs

```diff
@@ -289,6 +297,43 @@ impl Default for StoreType {
     }
 }
 
+/// This section creates a `create-reporting-task` Kubernetes Job, which enables the export of
+/// Prometheus metrics within NiFi.
+#[derive(Clone, Debug, Deserialize, JsonSchema, PartialEq, Serialize)]
+
+pub struct CreateReportingTaskJob {
```

Comment:
Otherwise docs etc wont match?
```suggestion
#[derive(Clone, Debug, Deserialize, JsonSchema, PartialEq, Serialize)]
#[serde(rename_all = "camelCase")]
pub struct CreateReportingTaskJob {
```

Reply:
Good catch, thanks!